### ISSUE TITLE
Bug: Recipient doesn't receive prompt to accept the file transfer

### ISSUE THREAD

**Issue by anishmittal2020** [cowbell/sharedrop#93]
In the help of <www.sharedrop.io>, it is mentioned that:

> Drag and drop a file directly on other person's avatar or click the avatar and select the file you want to send. The file transfer will start once the recipient accepts the file.

But, I have tried both ways the recipient doesn't gets the prompt to accept it. I am using the latest Chrome Browser on both Android 10 and Ubuntu 20.04.1 LTS x86_64.

**Comment by alimahedi** [comment]
yes it is not  working in my case I am using Windows 10 pc and POP OS! 
just showing to devices but not showing accept request on the recipient side when send file from sender.

![Screenshot (61)](https://user-images.githubusercontent.com/39080975/94112948-c8110f00-fe63-11ea-8896-753c15b86bcf.png)

**Comment by NoobTW** [comment]
Same here. Windows 10 & Ubuntu 21.04

**Comment by Mouadabdelghafouraitali** [comment]
Same issue here. The problem is 100% related to Windows 11, not Sharedrop, because it works when I test it with two phones but not when I test it with my PC and phone.
